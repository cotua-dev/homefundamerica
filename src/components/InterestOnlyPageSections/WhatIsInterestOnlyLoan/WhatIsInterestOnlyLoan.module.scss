section#whatIsInterestOnlyLoanSection {
    height: 40vh;

    @media screen and (max-width: 1000px) {
        height: 100%;
    }
}
