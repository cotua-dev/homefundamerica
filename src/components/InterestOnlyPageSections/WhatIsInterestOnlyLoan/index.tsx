import React, {
    FunctionComponent,
    ReactElement,
} from 'react';
import { whatIsInterestOnlyLoanSection } from './WhatIsInterestOnlyLoan.module.scss';
import * as sharedStyles from '../../sharedStyles/shared.module.scss';

export const WhatIsInterestOnlyLoan: FunctionComponent = (): ReactElement => {
    return (
        <section
            id={whatIsInterestOnlyLoanSection}
            className={sharedStyles.loanPositive}
        >
            <div className={sharedStyles.contentWrapper}>
                <h2>{`What Is an Interest-Only Mortgage?`}</h2>
                <p>
                    {`
                        An Interest-Only Mortgage is a home loan that
                        gives you the option to pay only the interest
                        on the principal amount for a set period of
                        time. After the interest-only term is over,
                        the payment converts to a principal-and-interest
                        payment that is fully amortized over the remaining
                        term of the mortgage.
                    `}
                </p>
                <p>
                    {`
                        Want additional information? After you apply, an
                        experienced Manhattan Banc Mortgage Specialist
                        will review all your options with you.
                    `}
                </p>
            </div>
        </section>
    );
}
