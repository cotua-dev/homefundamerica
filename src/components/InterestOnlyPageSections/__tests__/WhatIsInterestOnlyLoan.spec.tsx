import React from 'react';
import {
    render,
    RenderResult,
} from '@testing-library/react';
import { WhatIsInterestOnlyLoan } from '../WhatIsInterestOnlyLoan';

describe('WhatIsInterestOnlyLoan Component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<WhatIsInterestOnlyLoan/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});
