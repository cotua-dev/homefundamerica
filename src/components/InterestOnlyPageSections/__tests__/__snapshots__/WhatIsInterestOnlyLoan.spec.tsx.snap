// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WhatIsInterestOnlyLoan Component renders 1`] = `
<section
  id="whatIsInterestOnlyLoanSection"
>
  <div>
    <h2>
      What Is an Interest-Only Mortgage?
    </h2>
    <p>
      
                        An Interest-Only Mortgage is a home loan that
                        gives you the option to pay only the interest
                        on the principal amount for a set period of
                        time. After the interest-only term is over,
                        the payment converts to a principal-and-interest
                        payment that is fully amortized over the remaining
                        term of the mortgage.
                    
    </p>
    <p>
      
                        Want additional information? After you apply, an
                        experienced Manhattan Banc Mortgage Specialist
                        will review all your options with you.
                    
    </p>
  </div>
</section>
`;
